src/libdrakvuf/drakvuf.c

```diff
@@ -879,3 +879,47 @@ bool drakvuf_set_vcpu_gprs(drakvuf_t drakvuf, int vcpu, registers_t* regs)
 
     return xen_set_vcpu_ctx(drakvuf->xen, drakvuf->domID, vcpu, &ctx);
 }
+
+static bool is_valid_vcpu(drakvuf_t drakvuf, int vcpu)
```

Comment:
Any reason why the function takes `vcpu` as an `int`?

Reply:
Same as
```
bool drakvuf_set_vcpu_gprs(drakvuf_t drakvuf, int vcpu, registers_t* regs)
```

Reply:
Yea, I think that should also be an unsigned int :)

Reply:
ok I will fix this

Reply:
fixed, the rest is fixed with #1007